contracts/staking/query.go

```diff
@@ -77,3 +120,46 @@ func QueryValidators(t TxQueryHandler, from types.Address) ([]types.Address, err
 
 	return DecodeValidators(method, res.ReturnValue)
 }
+
+// DecodeBLSPublicKeys parses contract call result and returns arraty of bytes
+func DecodeBLSPublicKeys(
```

Comment:
Make this unexported. It's not used outside the package

Reply:
Fixed cb7ed389ef5c31b389767e802ddc46d032cfe2d2